nginx/Dockerfile-alpine

```diff
@@ -4,7 +4,7 @@ FROM nginx:${NGINX_VERSION}-alpine as build
 
 ARG MODSEC_VERSION=3.0.8
 
-# Note: pcre-dev (PCRE3) is required by the build description,
+# Note: pcre-dev (PCRE) is required by the build description,
```

Comment:
That's PCRE version 3, isn't it? The old version.
```suggestion
# Note: pcre-dev (PCRE 3) is required by the build description,
```

Reply:
We don't have version 3 yet.
PCRE is name of library.
libpcre3 is name of Debian-based package.

For more infomation:
- https://www.pcre.org/
- https://packages.debian.org/bullseye/libpcre3-dev
- https://en.wikipedia.org/wiki/Perl_Compatible_Regular_Expressions

Reply:
I'm aware of that. Version 3 of PCRE is the old, obsolete version. Version 2 is the current version. `libpcre3` is version 3. The note is important because we don't actually build with version 3 anymore but it is still required for the build script.

Reply:
PCRE doesn't have version 3 now. `libpcre3` is the name of the package, which contains PCRE1.

Reply:
You're right. The old version is indeed called PCRE1. The `3` is most likely the ABI version of `libpcre`. I did not know that. Thanks!

Reply:
I've updated my review comments to say `PCRE 1`.